docs/modules/internal_modules.md

```diff
@@ -0,0 +1,85 @@
+# List of Modules
+
+## What are internal modules?
+
+Internal modules just like regular modules, except that they run all the time. They do not have to be explicitly enabled. They can, however, be explicitly disabled if needed.
```

Comment:
Missing a word somewhere in there

Reply:
https://github.com/blacklanternsecurity/bbot/pull/1471/commits/76b10814ee4bb7ab6bc5e5cf48a9e9a4773f679c